src/components/players/validators.ts

```
@@ -322,3 +360,58 @@ export const validatePlayerUpdateRequest = () =>
+      optional: true,
+      custom: {
+        options: (val) =>
+          val === null || val === undefined || typeof val === "string", // Permite null, undefined o cadena
+        errorMessage: "tracking_number debe ser una cadena o null",
+      },
+    },
+    amount: {
+      in: ["body"],
+      optional: true,
+      custom: {
+        options: (val) => val === null || !isNaN(Number(val)),
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)